Specs/Scene/ModelSpec.js

```diff
@@ -265,7 +265,7 @@ defineSuite([
         expect(texturedBoxModel.ready).toEqual(true);
         expect(texturedBoxModel.asynchronous).toEqual(true);
         expect(texturedBoxModel.releaseGltfJson).toEqual(false);
-        expect(texturedBoxModel.cacheKey).toEndWith('Data/Models/Box-Textured/CesiumTexturedBoxTest.gltf');
+        expect(texturedBoxModel.cacheKey).not.toContain('Data/Models/Box-Textured/CesiumTexturedBoxTest.gltf');
```

Comment:
This isn't a very meaningful check.  I think `expect(texturedBoxModel.cacheKey).toBeDefined();` would be better

Reply:
Yes and added! I'll keep the old check as well to catch regressions. Sounds reasonable?